config/sync/views.view.content.yml

```diff
@@ -277,7 +277,7 @@ display:
         options:
           perm: 'access content overview'
       cache:
-        type: tag
+        type: none
```

Comment:
I wonder why this popped up. It seems to be related to this issue: https://www.drupal.org/project/content_lock/issues/3381841

In effect this will reduce caching on the content overview page and could affect the editor experience by reducing performance. As far as I can tell our editors have not complained about this problem so I wonder if it something that we actually want to do.

Consider discussing it with the business. On the other hand it might be hard to actively remove this behavior from the module which would leave us to keep it as is.

Reply:
>Consider discussing it with the business. On the other hand it might be hard to actively remove this behavior from the module which would leave us to keep it as is.

Well, if I manually remove it, won't it just get overwritten on next deploy?
E.g. content_lock might set it to `none`, but after we run `config-import`, it'll get set back to `tag`

Reply:
Okay, I understand what you mean now - the test fails if i do it like that, as the sites are remade from scratch 😅

Is it worth it to run it by the business then? Do we have any choice?

Reply:
> Do we have any choice?

Good question. If we want to reverse it we have to either introduce some hackery to remove the `deploy` hook for the module, use a reverse patch to remove it or the like. It will not be pretty but we could make it work.

Reply:
Fixed, as we talked about:

https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/2401/commits/cdd410b794a129a6244819bb6032ec56e6fc7f76

Reply:
and, i'm not too worried about the cache tag actually being an issue, especially not with the cypress test :)

Reply:
Fixed upstream: https://www.drupal.org/project/content_lock/issues/3525430